data/json/items/comestibles/carnivore.json

```diff
@@ -136,6 +136,31 @@
     "flags": [ "EATEN_HOT" ]
   },
   {
+    "id": "offal_pickled",
+    "copy-from": "offal",
+    "type": "COMESTIBLE",
+    "name": "pickled offal",
+    "name_plural": "pickled offal",
+    "description": "Cooked entrails, preserved in brine.  Packed with essential vitamins, tastes better than it smells.",
+    "parasites": 0,
+    "container" : "jar_glass",
+    "quench" : -1,
+    "fun": 0,
+    "stack_size" : 2
```

Comment:
Math goes like this:
* `stack_size` is a divisor on volume - actual volume is `volume / stack_size * 250ml`
* `charges` is a multiplier on charges things spawn with
* `result_mult` is another multiplier

So the final volume of an object is `charges * volume * result_mult / stack_size * 250ml`, which in this case looks like `1 * 1 * 2 / 2 * 250 ml = 250ml`, while the jar has capacity of 500ml.